samples/bluetooth/direction_finding_central/sample.yaml

```diff
@@ -0,0 +1,17 @@
+sample:
+  name: Direction Finding Central
+  description: Sample application showing central role of Direction Finding in connected mode
+tests:
+  sample.bluetooth.direction_finding_central:
+    harness: bluetooth
+    platform_allow: nrf52833dk_nrf52833
+    tags: bluetooth
+    integration_platforms:
+        - nrf52833dk_nrf52833
+  sample.bluetooth.direction_finding_central.aod:
+    extra_args: OVERLAY_CONFIG="overlay-aod.conf"
+    harness: bluetooth
+    platform_allow: nrf52833dk_nrf52833
```

Comment:
nRF53 DK too ? and nrf52833dk_nrf52820 ?

Reply:
Will add it in follow up PR.